opwen_email_server/mailers/connexion.py

```diff
@@ -0,0 +1,10 @@
+from opwen_email_server.action import ReceiveInboundEmail
```

Comment:
This looks like a typo and I believe is causing the CI to fail (see [build log](https://travis-ci.org/github/ascoderu/opwen-cloudserver/jobs/667904865#L3058)).

```suggestion
from opwen_email_server.actions import ReceiveInboundEmail
```

Ideally we'd catch this sort of issue during static analysis. I wonder if there's a linter that can check for unresolved modules. I thought that mypy would be doing this, but perhaps it's currently misconfigured.